apps/system/test/marionette/lib/utility_tray.js

```diff
@@ -64,7 +64,13 @@
     },
 
     get quickSettings() {
-      return this.client.helper.waitForElement(this.Selectors.quickSettings);
+      var element;
+      this.client.waitFor(function() {
+        element = this.client.findElement(this.Selectors.quickSettings);
+        return element && element.displayed();
+      }.bind(this));
+
+      return element;
```

Comment:
I saw https://bugzilla.mozilla.org/show_bug.cgi?id=1228753 error in some the runs I triggered, but didn't know that waitForElement was already checking if is visible, so let's keep it as it was. Thanks!